### DISCUSSION TITLE
Feature request: Increase volume by fixed increments when clicking to left or right of the slider(s)

### DISCUSSION THREAD

**Discussion by Gaurav-Original-ClassicShellTester** [File-New-Project/EarTrumpet#930] [Ideas]
In Windows XP, Vista and 7, if the Volume was 40, and I clicked above or below the slider (as indicated by red arrows), it would increase or decrease the volume exactly by 20. So it would be 60 or 20.

![image](https://user-images.githubusercontent.com/46422834/137213994-9fb4cd02-d852-489e-b0ac-5472b8a8b162.png)

This was easier to make significant differences in loudness without accurate clicking or positioning. The slider is already tiny to drag and with above behavior, I didn't have to precisely position over it, click it and drag it to exact new position. Just a click above or below was enough. Or we could click and hold to move it relatively fast in quick steps of 20. Or click fast, above or below, while still moving it in increments.

But the Windows 10 Volume control as well as EarTrumpet behave in an extremely non-optimal way:
![image](https://user-images.githubusercontent.com/46422834/137214256-c045521c-2df0-4144-8d0a-9a2eb2711d31.png)

From 34 it can go to 73 if I click on right somewhere and I don't know at what value I am clicking. It's supposed to go first to 54, then 74, 94. Or if I click to the left then 14.

My feature request is to implement the Windows XP/Vista/7/8.1 behavior into EarTrumpet, at least as an option, so all sliders behave that way. There is nothing to lose by implementing this feature because the user can always click and drag the slider if they want to jump from 34 to 73. But currently there is no way to move in increments of 20.

**Comment by riverar** [comment]
> There is nothing to lose by implementing this feature because the user can always click and drag the slider if they want to jump from 34 to 73.

You are suggesting that we change our slider behavior so it does not function like any other modern slider in Windows today. The loss is that users will expect this behavior and it will be missing/have changed. It's too late to make this change as proposed, however one possibility is to behave this when when a modifier key (e.g. shift) is held. I'll have to think about that some more.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Or a checkbox perhaps? "Change volume in increments when clicking to the left or right of the slider". There is one case where an option would be highly desirable for millions of past Windows users - since it was the default behavior for decades. 😊

**Reply by riverar** [reply]
@Gaurav-Original-ClassicShellTester That could work too, just more costly in terms of localization into 20+ languages.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
> That could work too, just more costly in terms of localization into 20+ languages.

Or a non-UI option? INI or Registry maybe so it doesn't need localization? A hotkey would be too cumbersome to hold down every time and to remember while changing volume.
